### PR TITLE
feat: add custom width

### LINKED ISSUE #4: Feature: Add width + Height params
Please add a parameter for width + height for larger ansi art, as the terminal size limits it and makes the project unusable as a library.

Here is where the width + height should be determined by the user: https://github.com/FHPythonUtils/AnsiToImg/blob/master/ansitoimg/render.py#L25-L28

Here's an example of the issue:

### PR DESCRIPTION
**Pull request by punctuations**
- closes #4

### CODE REVIEW

File: ansitoimg/render.py [hunk 1]
```diff
@@ -21,11 +21,11 @@
 TEXT_WIDTH = 12
 
 
-def _doRichRender(ansiText: str, wide: bool = True) -> Console:
-	console = Console(width=89 if wide else 49, record=True, file=StringIO())
+def _doRichRender(ansiText: str, wide: bool = True, width: int = 49) -> Console:
+	console = Console(width=round(width*1.82) if wide else width, record=True, file=StringIO())
```
Comment (FredHappyface) [inline ansitoimg/render.py:25]: Would you be happy to pull this out into a separate variable? round(width*1.82) if wide else width

File: ansitoimg/render.py [hunk 2]
```diff
@@ -127,20 +130,22 @@ async def _doGrabWebpage(url: str, resolution: tuple[int, int], fileName: str):
...
+def ansiToHTML(ansiText: str, fileName: str, theme: str | None = None, wide: bool = True, width: int = 49):
 	"""Convert an ANSI stream to a html file.
 
 	Args:
 		ansiText (str): ANSI text to convert
 		fileName (str): image file path
 		theme (str, optional): file path to base24 theme to use. Defaults to "onedark.yml".
 		wide (bool, optional): use a 'wide' terminal 89 vs 49 chars
+		width (int, optional): set the width for the image
+		height (int, optional): set the height for the image
```
Comment (FredHappyface) [inline ansitoimg/render.py:142]: height is not a param here

File: tests/data/fancyHelloWorld.html [hunk 3]
```diff
@@ -1,31 +1,31 @@
...
+</head>
+<html>
+<body>
+    <code>
+        <pre style="font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace">👋<span class="r1">Hello</span>, <span class="r2">World</span>🌏<span class="r3">!</span>
+<span class="r4">👋</span><span class="r5">🦄</span><span class="r6">🐘</span><span class="r7">13</span><span class="r8">3</span><span class="r9">7</span><span class="r10">🍄</span><span class="r11">🎃</span><span class="r12">🐦</span>
+</pre>
+    </code>
+</body>
+</html>
```
Comment (FredHappyface) [inline tests/data/fancyHelloWorld.html:31]: Do you mind sharing what OS you are on? Just curious as the rendered html is moderately different to that on WIndows

File: tests/data/metprintOutput.html [hunk 4]
```diff
@@ -1,158 +1,158 @@
...
+[<span class="r4">$  Deb</span>] debug
+[<span class="r10">* Info</span>] info
+[<span class="r11">+   Ok</span>] success
+[<span class="r12">/ Warn</span>] warning
+[<span class="r13">-  Err</span>] error
+[<span class="r9">! Crit</span>] critical
+</pre>
+    </code>
+</body>
+</html>
```
Comment (FredHappyface) [inline tests/data/metprintOutput.html:158]: No idea why there's a diff here + why the png output is borked for tests/data/metprintOutput.html.png. I wonder if this is something to do with line endings?

File: tests/simpletest.py [hunk 5]
```diff
@@ -37,28 +38,35 @@
...
 ansiToHTML(fancyHelloWorld, f"{THISDIR}/data/fancyHelloWorld.html", wide=False)
 ansiToHTML(fredHappyfaceHD, f"{THISDIR}/data/fredHappyfaceHD.html")
 ansiToHTML(metprintOutput, f"{THISDIR}/data/metprintOutput.html", wide=False)
+ansiToHTML(cat, f"{THISDIR}/data/cat.html", width=100)
 
 # To HTMLRender
 ansiToHTMLRender(fancyHelloWorld, f"{THISDIR}/data/fancyHelloWorld.html.png", wide=False)
 ansiToHTMLRender(fredHappyfaceHD, f"{THISDIR}/data/fredHappyfaceHD.html.png")
-ansiToHTMLRender(metprintOutput, f"{THISDIR}/data/metprintOutput.html.png", wide=False)
+ansiToHTMLRender(metprintOutput, f"{THISDIR}/data/cat.html.png", wide=False)
```
Comment (FredHappyface) [inline tests/simpletest.py:71]: This line looks off to me, can you revert this please?

**Review by FredHappyface — CHANGES_REQUESTED** [review]
Hi thank you so much for the PR!

Just got a couple minor comments if that's ok and then happy to merge. There's a couple questions out of curiosity too - those are not blocking any merging 

Thanks again for this - it's enormously appreciated :)

File: tests/simpletest.py (hunk 5 above)
Comment (punctuations) [reply tests/simpletest.py]: fixed in 27a58b7

**Comment by FredHappyface** [comment]
Hey cheers for the updates, can you run pre-commit over the code please as the indentation has been changed 

Thanks :)

File: tests/data/fancyHelloWorld.html (hunk 3 above)
Comment (FredHappyface) [reply tests/data/fancyHelloWorld.html:31]: Ah cheers :)

**Comment by punctuations** [comment]
> Hey cheers for the updates, can you run pre-commit over the code please as the indentation has been changed. You may need to pip install it.
> 
> ```shell
> pre-commit run -av
> ```
> 
>  
> Any pylint stuff can be ignored
> 
> Thanks :)

amended in [a1b1aee](https://github.com/FHPythonUtils/AnsiToImg/pull/5/commits/a1b1aee48d80d22e5bb0d6037cd81f40d0ea284f)

**Comment by FredHappyface** [comment]
Cheers, though it looks to still be using spaces

**Pull request closed without merging**